### DISCUSSION TITLE
How to re run fast-Dreambooth?

### DISCUSSION THREAD

**Discussion by BerryGoesNomad** [TheLastBen/fast-stable-diffusion#1633] [Q&A]
i run it before, everything goes smoothly, and then suddenly the connection is timed out
i tried to launch it again, follow every steps but it always fails

in Model download
it says "Skip this cell if you are loading a previous session that contains a trained model."
so i skipped that cell

in Dreambooth
i enter the last i used session name but it goes "Conversion error, if the error persists, remove the CKPT file from the current session folder"
![Screenshot_282](https://user-images.githubusercontent.com/101393609/221379789-09300806-8b53-428c-bf87-aee6ef6a2307.png)

and then i ignore that
so i go to next cell , in instance Images i run cell but i not upload anything because i uploaded the images last time i used , and then i go to next cell

when in Start DreamBooth
i press the box "resume training"  and then i start the cell , and i not changing anything 
but i got error
"Previous model not found, training a new model...
No model found, use the "Model Download" cell to download a model."

Please someone help me , what should i do?

**Comment by TheLastBen** [comment]
You are running a cpu runtime not GPU

**Reply by BerryGoesNomad** [reply]
how can i change it? can u tell me please?

**Reply by TheLastBen** [reply]
Runtime -> Change runtime type -> GPU

**Reply by BerryGoesNomad** [reply]
thats looks like i got to pay for the GPUs, right?

**Reply by TheLastBen** [reply]
google does offer free GPU a few hours a day

**Reply by BerryGoesNomad** [reply]
oh okay, thanks for helping bro
